src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -24,63 +25,67 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+/**
+ * This class describes a colony and contains all the data and methods for manipulating a Colony.
+ */
 public class Colony implements IColony
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 32 to the maximum authorized 30 or less. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1200)